<ui version="4.0" >
 <class>NMGConverterWidgetBase</class>
 <widget class="QWidget" name="NMGConverterWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QTreeWidget" name="listScripts" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>300</width>
           <height>490</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>300</width>
           <height>490</height>
          </size>
         </property>
         <column>
          <property name="text" >
           <string>Script name</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QTextBrowser" name="descBox" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>500</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>500</width>
             <height>100</height>
            </size>
           </property>
           <property name="autoFillBackground" >
            <bool>false</bool>
           </property>
           <property name="html" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select a script file in order to show it's description&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameScripts" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>500</width>
             <height>380</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>500</width>
             <height>5000</height>
            </size>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QTextBrowser" name="outputBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>811</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>811</width>
           <height>200</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="autoFillBackground" >
          <bool>false</bool>
         </property>
         <property name="html" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>715</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="runButton" >
           <property name="minimumSize" >
            <size>
             <width>91</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>91</width>
             <height>26</height>
            </size>
           </property>
           <property name="text" >
            <string>Run Script</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>216</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
